### PR TITLE
	battery percent setting

### PR DESCRIPTION
**Pull request by albertsal**
i add on preference the voice for setting  the threshold battery
notification discharge

and in hplus add the state battery for display notification

### CODE REVIEW

File: app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/AbstractDeviceSupport.java [hunk 1]
```diff
@@ -298,6 +298,10 @@ protected void handleGBDeviceEvent(GBDeviceEventBatteryInfo deviceEvent) {
         gbDevice.setBatteryLevel(deviceEvent.level);
         gbDevice.setBatteryState(deviceEvent.state);
 
+        Prefs prefs = GBApplication.getPrefs();
+        gbDevice.setBatteryThresholdPercent((short) prefs.getInt("battery_percent", 10));
```
Comment (cpfeiffer) [inline app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/AbstractDeviceSupport.java]: Can please you use a constant for all these occurrences of "battery_percent"?

**Comment by albertsal** [comment]
I have difficulty in understanding where to place the constant, I saw that there is a gbdevice of ...

**Comment by albertsal** [comment]
@cpfeiffer i do not understand why exists setBatteryThresholdPercent in gbdevice when unused.

if you agree I remove  getBatteryThresholdPercent  and setBatteryThresholdPercent from gbdevice and create a global object

**Comment by albertsal** [comment]
i change method

**Pull request closed without merging**